pkgs/top-level/all-packages.nix

```diff
@@ -41527,4 +41527,6 @@ with pkgs;
 
   wttrbar = callPackage ../applications/misc/wttrbar { };
 
+  gog-downloader = callPackage ../tools/games/gog-downloader { };
```

Comment:
The letter `g` doesn't come after the letter `w`.

Reply:
Neither it does two lines before:

```
waylyrics = callPackage ../applications/audio/waylyrics { };

gitrs = callPackage ../tools/misc/gitrs { };
```

This doesn't seem to be sorted alphabetically at all, contrary to the maintainer list where I put it in alphabetical order.

Reply:
Ok fair enough, let's leave it like that.